docs/products/opensearch/howto/migrate-opendistro-security-config-aiven.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Migrate Opendistro security configuration to Aiven for OpenSearch
+sidebar_label: Migrate Opendistro security configuration
+---
+
+Migrate your security configuration from an Opendistro service to Aiven for OpenSearch® using a migration script.
+
+The `.opendistro_security` index, which stores security settings, cannot be restored
+directly from an external snapshot. Instead, you use a migration script that interacts
+with the security REST API of both services.
+
+## What is migrated
+
+The migration script transfers the security configuration from the source Opendistro
+service to Aiven for OpenSearch. The following configurations are migrated:
+
+- Internal users (including passwords)
+- Action groups
+- Roles
+- Backend roles
+- Tenants
+
+The following configurations are **not** migrated:
+
+- Reserved, static, or hidden entries
+- Authentication methods and backend configurations, as these are configured differently
+  in Aiven for OpenSearch
+
+## Prerequisites
+
+Before starting the migration, ensure the following:
+
+- A machine with network access to both Opendistro and Aiven for OpenSearch services
+- Python 3.11 or higher installed
+- [Security management](/docs/products/opensearch/howto/enable-opensearch-security)
+  enabled on Aiven for OpenSearch
+- Admin user certificate and key from the source Opendistro service in PEM format
+- Opendistro security configuration version 2
+- HTTPS enabled on both services (the script does not verify server certificates)
+
+:::note
+
+- You need the admin user key and certificate to migrate internal users and their
+  passwords. The password hash is fetched directly from the `.opendistro_security` index
+  because the standard API endpoint doesn’t return it.
+- The script assumes HTTPS is enabled on both the source and target services.
+
+:::
+
+## Migration steps
+
+To migrate your security configuration from Opendistro to Aiven for OpenSearch:
+
+### Access the migration script
+
+The migration script is available in the [Aiven examples GitHub repository](https://github.com/aiven/aiven-examples/blob/main/solutions/migrate-opendistro-security-to-aiven-for-opensearch/avn-migrate-os-security-config.py).
+
+### Create the migration configuration file
+
+Create a JSON configuration file with the connection details for the source (Opendistro)
+and target (Aiven for OpenSearch) services. Following is an example:
+
+```json
+{
+  "source": {
+    "host": "source-ip-or-fqdn",
+    "port": source-port-number,
+    "key": "path-to-admin-key.pem",
+    "certificate": "path-to-admin-cert.pem"
+  },
+  "target": {
+    "host": "target-ip-or-fqdn",
+    "port": target-port-number,
+    "password": "os-sec-admin-user-password"
+  }
+}
+```
+
+### Run the migration script
+
+Once your configuration file is ready, run the following command to execute the
+migration:
```

Comment:
Let's never mention "the following commands", people know it is a command, and we have 2-4 words: 
```suggestion
Once your configuration file is ready, execute the migration:
```